src/webgl/p5.Geometry.js

```diff
@@ -185,17 +185,47 @@ p5.Geometry = class Geometry {
     return n.mult(Math.asin(sinAlpha) / ln);
   }
   /**
- * computes smooth normals per vertex as an average of each
- * face.
- * @method computeNormals
- * @chainable
- */
-  computeNormals() {
+   * computes smooth normals per vertex as an average of each
+   * face.
```

Comment:
Maybe we can add a sentence or two explaining how this works, so that users will know what to expect from it? I think the information would be:
- Normals are calculated for each face, and each vertex's normal is the average of its connected face normals
- Flat shading leaves duplicated vertices disconnected, while smooth shading reconnects duplicated vertices so their face normals average together (or some other way of explaining this difference that makes sense to you)